ci/test-coverage.sh

```diff
@@ -11,7 +11,11 @@ annotate() {
 source ci/upload-ci-artifact.sh
 source scripts/ulimit-n.sh
 
-scripts/coverage.sh
+scripts/coverage.sh "$@"
+
+if [[ -z $CI ]]; then
```

Comment:
For this manual use case: https://github.com/solana-labs/solana/pull/10589/commits/f9692ef4fc2314e8b7d9adea1992937752a272cf